packages/lesswrong/components/notifications/NotificationsPopover.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useCallback, useEffect, useRef, useState } from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import { Link } from "@/lib/reactRouterWrapper";
+import { HEADER_HEIGHT } from "../common/Header";
+import { useCurrentUser } from "../common/withUser";
+import { styles as popoverStyles } from "../common/FriendlyHoverOver";
+import { useNotificationDisplays } from "./NotificationsPage/useNotificationDisplays";
+import { karmaSettingsLink } from "./NotificationsPage/NotificationsPageFeed";
+import type { NotificationDisplay } from "@/lib/notificationTypes";
+import type { KarmaChanges } from "@/lib/collections/users/karmaChangesGraphQL";
+import type { KarmaChangeUpdateFrequency } from "@/lib/collections/users/schema";
+import { AnalyticsContext } from "@/lib/analyticsEvents";
+
+const notificationsSettingsLink = "/account?highlightField=auto_subscribe_to_my_posts";
```

Comment:
I couldn't see any way to link to a _section_ on the settings page, so I just linked to the first setting in the section.